include/inexor/vulkan-renderer/input/input.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+
+#include "inexor/vulkan-renderer/input/gamepad_data.hpp"
+#include "inexor/vulkan-renderer/input/keyboard_mouse_data.hpp"
```

Comment:
```suggestion
#include "inexor/vulkan-renderer/input/gamepad_data.hpp"
#include "inexor/vulkan-renderer/input/keyboard_mouse_data.hpp"

#include <GLFW/glfw3.h>
```

Reply:
Same: [rules for odering includes](https://inexor-vulkan-renderer.readthedocs.io/en/latest/development/engine-design/main.html#coding-style)